js/util/script_detection.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const verticalRegExp = new RegExp([
+    '[ᄀ-ᇿ]',
+    '[가-힣]',
+    '[ㄱ-ㆎ]',
+    '[一-鿌]',
+    '[㐀-䶵]',
+    // eslint-disable-next-line no-irregular-whitespace
+    '[　-〿]',
+    '\uD840[\uDC00-\uFFFF]|[\uD841-\uD872]|\uD873[\u0000-\uDEAF]', // '[𠀀-𬺯]'
+    '[！-￮]',
+    '[ぁ-ゟ]',
+    '[゠-ヿ]',
+    '[ㇰ-ㇿ]',
+    '[ꀀ-꓆]',
+    '[᠀-ᢪ]'
```

Comment:
@1ec5 makes sense. Only Inner Mongolia uses the traditional (vertical) Mongolian script, and all of those city names in Inner Mongolia are transliterated into Chinese characters. Mongolia the country uses Cyrillic (consistently LTR and horizontal).